services/app/src/components/chessBoard/Square.jsx

```diff
@@ -44,10 +46,10 @@ const Square = ({
       className={classNames("square", {
         'is-empty-tile': !items.length && row !== board.length - 1 && column !== 0
       })}
-      onMouseEnter={() => updatePopover(true)}
-      onMouseLeave={() => updatePopover(false)}
+      onMouseEnter={() => {updatePopover(true); setIsHovered(true)}}
+      onMouseLeave={() => {updatePopover(false); setIsHovered(false)}}
     >
-      {!!items.length && (
+      {(isHovered && (!items.length && row !== board.length - 1 && column !== 0) && (gameState === GameState.GAME_IN_PROGRESS)) && (
```

Comment:
Do we really need ```row !== board.length - 1 && column !== 0``` condition? From the looks of it its checked in the parent `Tile.jsx` component.